### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by JoeOster**
# Description
Going through all readme.md files for grammar issues. fixing, formatting issues, hyperlinks etc.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Sample Migration (Moving sample from old repository after completing checklist established)

### CODE REVIEW

File: DirectProgramming/DPC++/ParallelPatterns/loop-unroll/README.md [hunk 1]
```diff
@@ -41,16 +43,16 @@ This code sample is licensed under MIT license.
 The included header `dpc_common.hpp` is located at `%ONEAPI_ROOT%\dev-utilities\latest\include` on your development system.
 
 ### Running Samples in DevCloud
-If running a sample in the Intel DevCloud, remember that you must specify the compute node (fpga_compile or fpga_runtime) as well as whether to run in batch or interactive mode. For more information see the Intel® oneAPI Base Toolkit Get Started Guide ([https://devcloud.intel.com/oneapi/get-started/base-toolkit/](https://devcloud.intel.com/oneapi/get-started/base-toolkit/)).
+If running a sample in the Intel DevCloud, remember that you must specify the compute node (fpga_compile or fpga_runtime) and whether to run in batch or interactive mode. For more information see the Intel® oneAPI Base Toolkit Get Started Guide ([https://devcloud.intel.com/oneapi/get-started/base-toolkit/](https://devcloud.intel.com/oneapi/get-started/base-toolkit/)).
```
Comment (akertesz) [inline DirectProgramming/DPC++/ParallelPatterns/loop-unroll/README.md:46]: This is the version of loop_unroll for GPU. Might want to mention the GPU DevCloud nodes rather than just the FPGA ones.

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/Zlib_License.txt [hunk 2]
```diff
@@ -1,25 +0,0 @@
-zlib License
```
Comment (akertesz) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/Zlib_License.txt:1]: @JoeOster, why was this file deleted?

This isn't the boilerplate license file checked in with every code sample -- it's a 3rd party library dependence.

File: DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fast_recompile/README.md [hunk 3]
```diff
@@ -40,15 +40,15 @@ dpcpp <files.cpp> -o out.fpga -reuse-exe=out.fpga -Xshardware -fintelfpga
...
 
-If `out.fpga` is found but the compiler is unable to prove that the FPGA device code will yield a result identical to the last compilation, a warning is printed and the FPGA device code is fully recompiled. Since the compiler checks must be conservative, spurious recompilations can sometimes occur when using `-reuse-exe`.
+If `out.fpga` is found but the compiler cannot prove that the FPGA device code will yield a result identical to the last compilation, a warning is printed, and the FPGA device code is fully recompiled. Since the compiler checks must be conservative, spurious recompilations can sometimes occur when using `-reuse-exe`.
 
 ### Using the device link method
 
-The program accompanying this tutorial is separated into two files, `host.cpp` and `kernel.cpp`. Only the `kernel.cpp` file contains device code. 
+The program accompanying this tutorial is separated into two files, `host. cpp` and `kernel. cpp`. Only the `kernel. cpp` file contains device code.
```
Comment (akertesz) [inline DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fast_recompile/README.md:49]: While on the whole Grammarly has done a good job, it has done some really stupid things too. I've had to fix many. This one is particularly irksome and bears special mention.

File: DirectProgramming/DPC++FPGA/Tutorials/Tools/system_profiling/README.md [hunk 4]
```diff
@@ -66,23 +66,23 @@ void profiling_example(const std::vector<float>& vec_in,
...
 
 The Intercept Layer for OpenCL* Applications is an open-source tool that you can use to profile DPC++ designs at a system-level. Although it is not part of the oneAPI Base Toolkit installation, it is freely available on GitHub.
 
-This tool serves the following purpose:
-* Intercept host calls before they reach the device in order to gather performance data and log host calls.
-* Provide data to visualize the calls through time, and can separate them into *queued*, *submitted*, and *execution* sections for a better understanding of the execution.
-* Identify gaps (using visualization) in the runtime that may be leading to inefficient execution and throughput drops.
+This tool serves the following purposes:
+- Intercept host calls before they reach the device to gather performance data and log host calls.
+Provide data to visualize the calls through time and separate them into *queued*, *submitted*, and *execution* sections to understand the execution better.
```
Comment (akertesz) [inline DirectProgramming/DPC++FPGA/Tutorials/Tools/system_profiling/README.md:77]: Random deletion of bullet -- also very frustrating among the many fixes I'm having to make.

**Review by akertesz — APPROVED** [review]

**Comment by xmnboy** [comment]
@JoeOster -- changes to the `.../Tools/Migration/*` samples look fine.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md [hunk 5]
```diff
@@ -51,14 +54,14 @@ This code sample is licensed under MIT license.
 ###### **Stock TensorFlow**
 
 1. Create conda env: `$conda create -n stock-tensorflow python matplotlib ipykernel psutil pandas gitpython`
-2. Activate the created conda env: `$source activate stock-tensorflow`
+2. Activate the created conda env: `$source activate stock-tensorflow.`
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md:57]: no period '.' in the end.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md [hunk 6]
```diff
@@ -51,14 +54,14 @@ This code sample is licensed under MIT license.
...
 3. Install stock Tensorflow with a specific version: `(stock-tensorflow) $pip install tensorflow==2.2.0`
 4. Install extra needed package: `(stock-tensorflow) $pip install cxxfilt`
 5. Deactivate conda env: `(stock-tensorflow)$conda deactivate`
 6. Register the kernel to Jupyter NB: `$~/.conda/envs/stock-tensorflow/bin/python -m ipykernel install --user --name=stock-tensorflow`
 
 >  NOTE: Please change the python path if you have a different folder path for anaconda3.
-   After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall stock-tensorflow`.
+   After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall stock-tensorflow.`
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md:64]: no period '.' in the end of the command.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md [hunk 7]
```diff
@@ -68,12 +71,12 @@ This code sample is licensed under MIT license.
...
 2. Create conda env: `$conda create --name intel-tensorflow --clone tensorflow`
 3. Activate the created conda env: `$source activate intel-tensorflow`
-4  Install extra needed package: `(intel-tensorflow) $pip install cxxfilt matplotlib ipykernel psutil pandas gitpython`
+4  Install the extra needed package: `(intel-tensorflow) $pip install cxxfilt matplotlib ipykernel psutil pandas gitpython`
 5. Deactivate conda env: `(intel-tensorflow)$conda deactivate`
 6. Register the kernel to Jupyter NB: `$~/.conda/envs/intel-tensorflow/bin/python  -m ipykernel install --user --name=intel-tensorflow`
 
 > NOTE: Please change the python path if you have a different folder path for anaconda3. 
-  After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall intel-tensorflow`.
+  After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall intel-tensorflow.`
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md:79]: same as above.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md [hunk 8]
```diff
@@ -88,19 +91,19 @@ This code sample is licensed under MIT license.
 6. Register the kernel to Jupyter NB: `$~/anaconda3/envs/stock-tensorflow/bin/python  -m ipykernel install --user --name=stock-tensorflow`
 
 > NOTE: Please change the python path if you have a different folder path for anaconda3. 
-  After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall stock-tensorflow`.
+  After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall stock-tensorflow.`
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md:94]: same as above.

File: AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md [hunk 9]
```diff
@@ -88,19 +91,19 @@ This code sample is licensed under MIT license.
...
 2. Create conda env: `$conda create --name intel-tensorflow --clone tensorflow`
 3. Activate the created conda env: `$source activate intel-tensorflow`
-4  Install extra needed package: `(intel-tensorflow) $pip install cxxfilt matplotlib ipykernel psutil pandas gitpython`
+4  Install the extra needed package: `(intel-tensorflow) $pip install cxxfilt matplotlib ipykernel psutil pandas gitpython`
 5. Deactivate conda env: `(intel-tensorflow)$conda deactivate`
 6. Register the kernel to Jupyter NB: `$~/anaconda3/envs/intel-tensorflow/bin/python  -m ipykernel install --user --name=intel-tensorflow`
 
 > NOTE: Please change the python path if you have a different folder path for anaconda3. 
-  After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall intel-tensorflow`.
+  After profiling, users can remove the kernel from Jupyter NB with `$jupyter kernelspec uninstall intel-tensorflow.`
```
Comment (louie-tsai) [inline AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_PerformanceAnalysis/README.md:106]: same here.
